src/components/settings/account/AccountDashboard.tsx

```diff
@@ -6,7 +6,7 @@ import { H1, H2 } from '../../ui/headline';
 
 import Loader from '../../ui/Loader';
 import Button from '../../ui/button';
-import Infobox from '../../ui/infobox';
+import Infobox from '../../ui/infobox/index';
```

Comment:
Can we also delete the other infobox.js file?

Reply:
ok will check and update this PR

Reply:
Lets take this up as a separate PR. Approving and merging this one